app/data/models/Card.py

```diff
@@ -0,0 +1,16 @@
+class Card:
+
+    def __init__(self, card_id, deck_id, word, translation ):
+        self.card_id = card_id
+        self.deck_id = deck_id
+        self.word = word
+        self.translation = translation
+
+
+    @classmethod
+    def new_card(cls, deck_id, word, translation):
```

Comment:
whats cls?

Reply:
I'm not gonna lie :) don't understand it fully, will have to read more about it. cls is an uninstantiated class passed as an argument. What I know it supports static factory pattern, should I dive into it?

Reply:
yes i guess check to know whats that is. Looks like equivalent of pointer to method in C.